### PR TITLE
Maoh/add components

### PR DESCRIPTION
**Pull request by MaohDesign**
The UI of the Explore page has been updated.
SP support will be done separately.

### CODE REVIEW

**Comment by wildmouse** [comment]
I got a type error for `yarn react-app:typecheck`. 

Seems you should run `yarn add @material-ui/lab`? 

```
src/components/atoms/SelectTokenInput/index.tsx:19:26 - error TS2307: Cannot find module '@material-ui/lab/Autocomplete' or its corresponding type declarations.

19 import Autocomplete from "@material-ui/lab/Autocomplete";
                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

src/components/atoms/SelectTokenInput/index.tsx:53:26 - error TS7006: Parameter 'option' implicitly has an 'any' type.

53         getOptionLabel={(option) => option.tokenName}
                            ~~~~~~

src/components/atoms/SelectTokenInput/index.tsx:56:23 - error TS7006: Parameter 'params' implicitly has an 'any' type.

56         renderInput={(params) => (
                         ~~~~~~

src/components/molecules/TokenInfoBar/index.tsx:25:22 - error TS2307: Cannot find module '@material-ui/lab/Skeleton' or its corresponding type declarations.

25 import Skeleton from "@material-ui/lab/Skeleton";
                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: frontend/packages/react-app/src/components/organisms/TokenCampaignDetail/index.tsx [hunk 1]
```diff
@@ -17,60 +17,102 @@
...
+            Distributor Type:
+          </Typography>
+          <Typography variant="body1" style={{ paddingLeft: 8 }}>
+            {distributorType}
+          </Typography>
+        </Box>
+        <Box>
+          {/* //TODO: ADD LINK */}
+          <Link
+            href="#"
```
Comment (wildmouse) [inline frontend/packages/react-app/src/components/organisms/TokenCampaignDetail/index.tsx:104]: This one is better than no contents? 
https://medium.com/iroiro-social-token/introducing-how-to-distribute-token-by-wallet-address-or-url-4e4646581047

**Comment by wildmouse** [comment]
Seems `TokenBasicInformationTemplate` can be deleted?

File: frontend/packages/react-app/src/components/templates/ExplorePageTemplate/index.stories.tsx [hunk 2]
```diff
@@ -28,37 +30,20 @@ export default {
...
-    tokens: [],
-    isOpen: false,
-    inputTokenAddress: "",
-    tokenAddress: "",
-    type: "explore",
-    color: "primary",
-  },
-};
-
 export const Loading = Template.bind({});
```
Comment (wildmouse) [inline frontend/packages/react-app/src/components/templates/ExplorePageTemplate/index.stories.tsx:49]: Should loading story display a skeleton component?

**Comment by wildmouse** [comment]
A warning is displayed on console like following: 

```
react_devtools_backend.js:2430 
The component styled.p with the id of "sc-khAkjo" has been created dynamically.
You may see this warning because you've called styled inside another component.
To resolve this only create new StyledComponents outside of any render method and function component. 
    at MenuButton (http://localhost:3000/static/js/main.chunk.js:995:3)
    at div
    at styled.div (http://localhost:3000/static/js/1.chunk.js:212606:6)
    at div
    at div
    at StyledComponent (http://localhost:3000/static/js/1.chunk.js:267080:28)
    at div
    at ExplorePageTemplate (http://localhost:3000/static/js/main.chunk.js:11188:3)
    at ExplorePage (http://localhost:3000/static/js/main.chunk.js:9387:76)
    at Route (http://localhost:3000/static/js/1.chunk.js:171070:29)
    at Switch (http://localhost:3000/static/js/1.chunk.js:171272:29)
    at TokenProvider (http://localhost:3000/static/js/main.chunk.js:12387:3)
    at div
    at Router (http://localhost:3000/static/js/1.chunk.js:170705:30)
    at HashRouter (http://localhost:3000/static/js/1.chunk.js:170370:35)
    at div
    at App
    at Web3ReactProvider (http://localhost:3000/static/js/1.chunk.js:35461:27)
    at ThemeProvider (http://localhost:3000/static/js/1.chunk.js:266011:24)
    at ApolloProvider (http://localhost:3000/static/js/1.chunk.js:216866:19)
```

**Comment by wildmouse** [comment]
`TabMenuForFunPage` => `TabMenuForFanPage`

File: frontend/packages/react-app/src/components/organisms/TokenCampaignDetail/index.tsx (hunk 1 above)
Comment (MaohDesign) [reply frontend/packages/react-app/src/components/organisms/TokenCampaignDetail/index.tsx:104]: I was looking for that!

**Comment by MaohDesign** [comment]
> I got a type error for `yarn react-app:typecheck`. 
> 
> 
> 
> Seems you should run `yarn add @material-ui/lab`? 
> 
> 
> 
> ```
> 
> src/components/atoms/SelectTokenInput/index.tsx:19:26 - error TS2307: Cannot find module '@material-ui/lab/Autocomplete' or its corresponding type declarations.
> 
> 
> 
> 19 import Autocomplete from "@material-ui/lab/Autocomplete";
> 
>                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
> 
> 
> 
> src/components/atoms/SelectTokenInput/index.tsx:53:26 - error TS7006: Parameter 'option' implicitly has an 'any' type.
> 
> 
> 
> 53         getOptionLabel={(option) => option.tokenName}
> 
>                             ~~~~~~
> 
> 
> 
> src/components/atoms/SelectTokenInput/index.tsx:56:23 - error TS7006: Parameter 'params' implicitly has an 'any' type.
> 
> 
> 
> 56         renderInput={(params) => (
> 
>                          ~~~~~~
> 
> 
> 
> src/components/molecules/TokenInfoBar/index.tsx:25:22 - error TS2307: Cannot find module '@material-ui/lab/Skeleton' or its corresponding type declarations.
> 
> 
> 
> 25 import Skeleton from "@material-ui/lab/Skeleton";
> 
>                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~
> 
> ```

It looks like there was a lot of stuff going on and I had to delete some code changes. @material-ui/lab is necessary. Thank you!

**Comment by MaohDesign** [comment]
> Seems `TokenBasicInformationTemplate` can be deleted? 

Can I delete all components that it don't be needed?

**Comment by MaohDesign** [comment]
> A warning is displayed on console like following: 
> 
> 
> 
> ```
> 
> react_devtools_backend.js:2430 
> 
> The component styled.p with the id of "sc-khAkjo" has been created dynamically.
> 
> You may see this warning because you've called styled inside another component.
> 
> To resolve this only create new StyledComponents outside of any render method and function component. 
> 
>     at MenuButton (http://localhost:3000/static/js/main.chunk.js:995:3)
> 
>     at div
> 
>     at styled.div (http://localhost:3000/static/js/1.chunk.js:212606:6)
> 
>     at div
> 
>     at div
> 
>     at StyledComponent (http://localhost:3000/static/js/1.chunk.js:267080:28)
> 
>     at div
> 
>     at ExplorePageTemplate (http://localhost:3000/static/js/main.chunk.js:11188:3)
> 
>     at ExplorePage (http://localhost:3000/static/js/main.chunk.js:9387:76)
> 
>     at Route (http://localhost:3000/static/js/1.chunk.js:171070:29)
> 
>     at Switch (http://localhost:3000/static/js/1.chunk.js:171272:29)
> 
>     at TokenProvider (http://localhost:3000/static/js/main.chunk.js:12387:3)
> 
>     at div
> 
>     at Router (http://localhost:3000/static/js/1.chunk.js:170705:30)
> 
>     at HashRouter (http://localhost:3000/static/js/1.chunk.js:170370:35)
> 
>     at div
> 
>     at App
> 
>     at Web3ReactProvider (http://localhost:3000/static/js/1.chunk.js:35461:27)
> 
>     at ThemeProvider (http://localhost:3000/static/js/1.chunk.js:266011:24)
> 
>     at ApolloProvider (http://localhost:3000/static/js/1.chunk.js:216866:19)
> 
> ```

It's going to take some time.🤔I'd like to put it off for a bit.

**Comment by wildmouse** [comment]
> > Seems `TokenBasicInformationTemplate` can be deleted?
> 
> Can I delete all components that it don't be needed?

Yes, if its not used anymore.

**Review by wildmouse — APPROVED** [review]
LGTM 🏃‍♂️

**Pull request merged**